test/indexer/test_hca_indexer.py

```diff
@@ -397,6 +399,21 @@ def test_project_contact_extraction(self):
             self.assertEqual({'Human Cell Atlas wrangler', None},
                              contributor_values['project_role'])
 
+    def test_diseases_field(self):
+        """
+        Index a bundle with a specimen `diseases` value that is differs from its donor `diseases` value
+        and assert that only the specimen's `diseases` value is in the indexed document.
+        """
+        self._mock_index(self.disease_bundle)
+
+        es_results = self._get_es_results()
+        for index_results in es_results:
+            bundle = index_results['_source']['bundles'][0]
+            if 'contents' in bundle:
+                self.assertEqual(1, len(bundle['contents']['specimens'][0]['disease']))
+                specimen_disease = bundle['contents']['specimens'][0]['disease'][0]
```

Comment:
The point of extracting a variable is to avoid duplicating an expression. The way you did this, the expression still occurs twice.

Reply:
https://en.wikipedia.org/wiki/Don%27t_repeat_yourself